pymws/feeds.py

```diff
@@ -54,6 +57,7 @@ def get_feed_submission_list(self, **kwargs):
 
         `Learn more <https://docs.developer.amazonservices.com/en_US/feeds/Feeds_GetFeedSubmissionList.html>`_
         """    # noqa: E501
+        flatten_list(kwargs, 'FeedSubmissionIdList', 'Id')
```

Comment:
can you also flatten FeedType

![image](https://user-images.githubusercontent.com/305807/93884842-6ad55c00-fcb1-11ea-9367-e9224e366cc8.png)